phpunit/style-engine/class-wp-style-engine-gutenberg-test.php

```diff
@@ -0,0 +1,156 @@
+<?php
+/**
+ * Tests the Style Engine class and associated functionality.
+ *
+ * @package    Gutenberg
+ * @subpackage block-library
```

Comment:
Tiniest of nits: once we've moved the style engine class in a follow-up, this'll be:

```suggestion
 * @subpackage style-engine
```

(We can address that in the follow-up PR if you'd like)

Reply:
Ctrl-C Ctrl-V 💥